README.md

```diff
@@ -20,7 +20,39 @@ var morgan = require('morgan')
 
 Create a new morgan logger middleware function using the given `format` and `options`.
 The `format` argument may be a string of a predefined name (see below for the names),
-a string of a format string, or a function that will produce a log entry.
+a string of a format string, or a format function that will produce a log entry.
+
+Format functions accept three arguments `tokens`, `req`, and `res` where `tokens` represents
+an object with all known tokens. It is expected to return a string. For the sake of example, the
+following four code snippets are equivalent.
+
+##### Using a [predefined format string](#predefined-formats)
+```
+morgan(':tiny')
+```
+
+##### Using format string of [predefined tokens](#tokens)
+```js
+morgan(':method :url :status :res[content-length] - :response-time ms')
+```
+
+##### Using a format function [from `morgan.compile`](#morgancompileformat)
+``` js
+morgan(morgan.compile(':method :url :status :res[content-length] - :response-time ms'))
```

Comment:
Sure. Changed this in [1c9416c](https://github.com/expressjs/morgan/pull/125/commits/1c9416c7692b359b2f4bcfe7b9b357fe14dc9306). Not sure why this Github comment wasn't hidden automatically because the line was changed.